src/components/drawer/drawer.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export interface DrawerPropTypes {
+  children: React.FunctionComponent | React.ComponentClass;
+  defaultExpanded?: boolean;
+  expanded?: boolean;
+  onChange?: () => void;
```

Comment:
We're missing the callback signature here `e` & `isExpanded`.

Reply:
added signature